cmd/logcli/config.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/grafana/loki/pkg/helpers"
+)
+
+// Config is the root config for Logcli.
+type Config struct {
+	Addr     string `yaml:"addr,omitempty"`
+	Username string `yaml:"username,omitempty"`
+	Password string `yaml:"password,omitempty"`
+}
+
+func getConfig(configFile string) *Config {
+	if configFile == "" {
+		home := os.Getenv("HOME")
+		configFile = fmt.Sprintf("%s/%s", home, "logcli-config.yaml")
+	}
+
+	config := Config{"https://log-us.grafana.net", "", ""}
+
+	if err := helpers.LoadConfig(configFile, &config); err != nil {
```

Comment:
I think I'd prefer if this function returned `(*Config, error)` and the called dealt with logging errors.